src/ga4gh/vrs/extras/translator.py

```diff
@@ -91,11 +91,17 @@ def _ir_stype(a):
         """Get accession's sequence type"""
         if a.startswith("refseq:NM_"):
             return "n"
+        if a.startswith("refseq:NR_"):
+            return "n"
         if a.startswith("refseq:NP_"):
             return "p"
+        if a.startswith("refseq:NC_012920"):
+            return "m"
+        if a.startswith("refseq:NW_"):
+            return "g"
         if a.startswith("refseq:NG_"):
             return "g"
-        if a.startswith("refseq:NC_"):
+        if a.startswith("refseq:NC_00"):
```

Comment:
@larrybabb Is it necessary to include the `00`?

```suggestion
        if a.startswith("refseq:NC_"):
```